src/app/integrations/list/list.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'patternfly-sass/assets/stylesheets/patternfly/color-variables';
+@import '../../../../node_modules/patternfly-sass/assets/stylesheets/patternfly/color-variables';
```

Comment:
If we bring in this import we should see what color variables could be used in the file.

Reply:
That was the general idea of the `/src/scss/_common.scss` file, or a `main.scss` file, which I think we used to have, but no idea what happened to it. I could add it back in.